aisera/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ed21d816-fc99-4454-99d9-da6404d448a7",
+  "app_id": "aisera_aiops",
+   "assets": {
+    "integration": {
+      "source_type_name": "aisera",
+      "configuration": {}
+    }
+  },
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Aisera Aiops.",
```

Comment:
Please provide more details here on what the integration delivers to Datadog (80 character limit)